README.md

```diff
@@ -77,17 +77,12 @@ of the Azure Active Directory window respectively as *Name* and *Directory ID*
 
 #### Register the client app (MyDirectorySearcherApp)
 
-1. In the  **Azure Active Directory** pane, click on **App registrations** and choose **New application registration**.
+1. In the  **Azure Active Directory** pane, click on **App registrations (Preview)** and choose **New application**.
 1. Enter a friendly name for the application, for example 'MyDirectorySearcherApp' and select 'Native' as the *Application Type*.
 1. For the *Redirect URI*, enter `https://<your_tenant_name>/MyDirectorySearcherApp`, replacing `<your_tenant_name>` with the name of your Azure AD tenant.

```

Comment:
Actually,  things have evolved since this sample was created. The following URI might be better `https://login.microsoftonline.com/common/oauth2/nativeclient`. (for which the developer can check a check-box in the portal). But no obligation here.